sql/core/src/test/scala/org/apache/spark/status/protobuf/sql/KVStoreProtobufSerializerSuite.scala

```diff
@@ -48,6 +48,43 @@ class KVStoreProtobufSerializerSuite extends SparkFunSuite {
     assert(result.metricValues == input.metricValues)
   }
 
+  test("SQLExecutionUIData with null string fields") {
```

Comment:
why don't we just update the previous test case with a new loop for null values. Or, we can abstract the code for checking whether two SQLExecutionUIData are the same.

Reply:
[5bb20d9](https://github.com/apache/spark/pull/39682/commits/5bb20d90ce0e17a999423be3d6577b7835f1aaff) move this to previous test case with a new loop